### PR TITLE
Make .dockerignore match the what the Dockerfile uses to copy the binary

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

**Comment by geoand** [comment]
This follows up what I wrote [here](https://github.com/quarkusio/quarkus/pull/1407#issuecomment-471889988)

File: devtools/common/src/main/resources/templates/dockerignore.ftl [hunk 1]
```diff
@@ -1,2 +1,2 @@
 *
-!target/${project_artifactId}
\ No newline at end of file
+!target/*-runner
```
Comment (cescoffier) [inline devtools/common/src/main/resources/templates/dockerignore.ftl]: @geoand does it make sense to to -runner and -runner.jar

**Review by cescoffier — APPROVED** [review]

File: devtools/common/src/main/resources/templates/dockerignore.ftl (hunk 1 above)
Comment (geoand) [reply devtools/common/src/main/resources/templates/dockerignore.ftl]: @cescoffier That is what I am doing in #1312 :wink: . Would you like me to add it in this PR and get it over with?

Comment (cescoffier) [reply devtools/common/src/main/resources/templates/dockerignore.ftl]: Yes, let's do it once for all.

Comment (geoand) [reply devtools/common/src/main/resources/templates/dockerignore.ftl]: Makes sense! Fixed

**Comment by anthonydahanne** [comment]
ah , I just focused on the executable binary, not the jars !

**Comment by geoand** [comment]
@anthonydahanne No worries :)

**Pull request merged**